app/web/pages/gallery/[galleryId].tsx

```diff
@@ -0,0 +1,59 @@
+import { Container } from "@mui/material";
+import { appGetLayout } from "components/AppRoute";
+import HtmlMeta from "components/HtmlMeta";
+import PageTitle from "components/PageTitle";
+import GalleryEditor from "features/gallery/GalleryEditor";
+import { useTranslation } from "i18n";
+import { GLOBAL } from "i18n/namespaces";
+import { GetStaticPaths, GetStaticProps } from "next";
+import { useRouter } from "next/router";
+import nextI18nextConfig from "next-i18next.config";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+export const getStaticPaths: GetStaticPaths = () => ({
+  paths: [],
+  fallback: "blocking",
+});
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => ({
+  props: {
+    ...(await serverSideTranslations(
+      locale ?? "en",
+      [GLOBAL],
```

Comment:
Again, I'm not sure we wanna do global here. Since it's a new service should it have it's own "gallery" translations file?

I assume we will use these components eventually in the "My Home" section too so could make sense.